src/rpft/rapidpro/models/routers.py

```
@@ -537,8 +542,8 @@ def validate(self):
             raise ValueError(f'Invalid router test type: "{self.type}"')
         if not RouterCase.TEST_VALIDATIONS[self.type](self.arguments):
             print(
```

We should use the logger, or raise an exception, rather than print.